specs/commandline/policy.md

```diff
@@ -0,0 +1,671 @@
+
+An example of `trustpolicy.json`:
+
+```jsonc
+{
+    "version": "1.0",                                                   // version info
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-dev",                              // Name of the 1st policy
+            "registryScopes": [ "dev.wabbitnetworks.io/net-monitor" ],  // The repository list that policy applies to
+            "signatureVerification": {                                  // The level of verification - strict, permissive, audit, skip
+                "level": "strict"
+                "override" : {
+                     "expiry" : "log",
+                     "authenticity": "log"
+                }
+            },
+            "trustStores": [ "ca:wabbit-networks-dev" ],                // The trust stores that contains the X.509 certificates
+            "trustedIdentities": [                                      // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+            ]
+        },
+        {
+            "name": "wabbit-networks-prod",                             // Name of the 2nd policy.
+            "registryScopes": [ "prod.wabbitnetworks.io/net-monitor" ],       
+            "signatureVerification": {                                
+                "level": "permissive"
+            },
+            "trustStores": [ "ca:wabbit-networks-prod" ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Prod, CN=wabbit-networks.io"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  add           Add trust policies.
+  delete        Delete trust policies.
+  list          List trust policies.
+  update        Update trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy add
+
+```text
+Add trust policies.
+
+Usage:
+  notation policy add [flags] <policy_name>
+
+Flags:
+      --cert-file          string   optional certificate filepath
+      --custom-level       string   optional custom level to based existing verification level, e.g. "authenticity=log,expiry=log"
+  -h, --help                        help for add
+      --id                 string   optional trust identity aka certificate subject info
+      --repo               string   optional repository URI (default "*")
+      --trust-store        string   required trust store in format "<trust_store_type>:<trust_store_name>", e.g. "ca:my_trust_store"
```

Comment:
Same here. It's an array.
```suggestion
      --trust-store        stringArray   required trust stores in format <trust_store_type>:<trust_store_name>
```
examples can be moved into the `Long` message of the command.